frontend/src/components/UserTables/StudentsTable.tsx

```diff
@@ -1,13 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
+import cn from 'classnames';
+import { useReq } from '../../context/req';
 import { Row, Table } from '../TableComponents/TableComponents';
 import { useRiders } from '../../context/RidersContext';
+import styles from './table.module.css';
+
 
 const StudentsTable = () => {
   const { riders } = useRiders();
   const history = useHistory();
+  const { withDefaults } = useReq();
   const colSizes = [1, 0.75, 0.75, 1.25, 1];
-  const headers = ['Name / NetId', 'Number', 'Address', 'Duration', 'Disability'];
+  const headers = ['Name / NetId', 'Number', 'Address', 'Usage', 'Disability'];
+  const [usage, setUsage] = useState({ studentRides: 0, noShowCount: 0 });
+  const getUsageData = (id: string) => {
+    fetch(`/api/riders/${id}/usage`, withDefaults())
+      .then((res) => (res.json()))
+      .then((data) => setUsage(data));
+    return {
+      data:
+        <div className={styles.usage}>
+          <span className={styles.usageContainer}>
+            <span className={cn(styles.ridesCount, styles.usageTag)}></span>
+            {usage.studentRides} Rides
+          </span>
+          <span className={styles.usageContainer}>
+            <span className={cn(styles.noShow, styles.usageTag)}></span>
+            {usage.noShowCount} No Shows
+          </span>
+        </div>,
+    };
+  };
```

Comment:
just found a bug: usage is the state for the whole table, not just an individual rider. so every rider has the same usage at the moment. we'd either need to make a request each per user (which would be expensive) and figure out a way to keep track of that in state, or create another endpoint to get the usage of all riders in the table. if you have time id like you to try the second option, if not i can try to do it within the next couple days

Reply:
i may have some time to work on it this weekend but I don't think I'll be able to finish it by the work session :(